ocpmodels/datasets/oc22_lmdb_dataset.py

```diff
@@ -155,15 +154,15 @@ def __getitem__(self, idx):
         # convert s2ef energies to raw energies
         if attr == "y":
             # OC20 data
-            if "oc22" not in data_object and self.train_total:
+            if "oc22" not in data_object and self.oc20_ref:
```

Comment:
Same comment as above. It seems like a non-obvious assumption that someone wants to train on total energies if they have the `oc20_ref` flag specified. `total_energy: True|False` is much more explicit.